### PR TITLE
chore: enable new fee tier on all chain

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: src/hooks/useProAmmPreviousTicks.ts [hunk 1]
```diff
@@ -83,49 +83,41 @@ export function useProAmmMultiplePreviousTicks(
...
-            console.debug('Failed to get last 24h data, maybe position is just created', e)
+          ?.getTotalFeesOwedToPosition(NETWORKS_INFO[chainId].elastic.nonfungiblePositionManager, poolAddress, tokenID)
+          .then((res: { token0Owed: BigNumber; token1Owed: BigNumber }) => {
+            setFee([res.token0Owed.toString(), res.token1Owed.toString()])
           })
+          .finally(() => setLoading(false))
+      }
+
+      getFee()
+      i = window.setInterval(() => getFee(), 6_969)
```
Comment (namgold) [inline src/hooks/useProAmmPreviousTicks.ts:108]: what the heo 6969

Comment (viet-nv) [reply src/hooks/useProAmmPreviousTicks.ts:108]: just a number

File: src/utils/formatTickPrice.ts [hunk 2]
```diff
@@ -17,5 +17,5 @@ export function formatTickPrice(
   if (!price && placeholder !== undefined) {
     return placeholder
   }
-  return formatPrice(price, 6)
+  return formatPrice(price, 5)
```
Comment (namgold) [inline src/utils/formatTickPrice.ts:20]: any specific reason?

**Review by namgold — APPROVED** [review]

File: src/utils/formatTickPrice.ts (hunk 2 above)
Comment (viet-nv) [reply src/utils/formatTickPrice.ts:20]: for consistent with left price and right price
lowest feetier has tick spacing 1 -> that means price move 0.0001 each tick

**Review by namgold — APPROVED** [review]

**Pull request merged**